src/services/users/get-invited-user-detail.test.js

```diff
@@ -0,0 +1,61 @@
+const { expect } = require('chai')
+const { faker } = require('@faker-js/faker')
+const db = require('../../db')
+const {fixtures} = require('../../../test/utils')
+const getInvitedUserDetail = require('./get-invited-user-detail')
+
+
+describe('GET invited-user information', () => {
```

Comment:
The `GET` in capital letters makes it look like an api test to me. Just me worrying you

```suggestion
describe('Get invited user information', () => {
```

Reply:
that's true